starmark.dev/e2e/storage-connector.spec.ts

```
@@ -0,0 +1,276 @@
+    
+    // Wait for success message
+    const successMessage = page.getByTestId('success-message');
+    await expect(successMessage).toBeVisible({ timeout: 10000 });
+    
+    // Verify API response structure
+    expect(apiResponse).toBeDefined();
+    expect(apiResponse).toHaveProperty('success', true);
+    expect(apiResponse).toHaveProperty('id');
+    expect(apiResponse).toHaveProperty('message');
+    expect(apiResponse).toHaveProperty('metadata');
+    expect(apiResponse!.metadata).toHaveProperty('connectorsUsed');
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)